srml/council/src/motions.rs

```diff
@@ -263,6 +263,7 @@ impl<
 	AccountId
 > EnsureOrigin<O> for EnsureMember<AccountId> {
 	type Success = AccountId;
+	type Error = &'static str;
```

Comment:
shouldn't this be generic in case we want to use this with a structured `Error`?